_release/sync-rtc.cron

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-exec /usr/bin/sync-rtc
+exec /usr/bin/rtc write
```

Comment:
Should this be write-if-synced?

Reply:
Also can you pipe the output to logger (`| logger [opts]`) Otherwise the output gets lost.